.github/workflows/email.yml

```diff
@@ -36,8 +36,8 @@ jobs:
          
           
       - name: Send mail  
-       
-        if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true  
```

Comment:
github.event_name == 'pull_request' is redundant so removed it. On: Push in the begining of the workflow checks for the event name so removed it.
github.event.action == 'closed' may not be true as soon as the PR is merged so removed it.